include/conversation.php

```diff
@@ -901,7 +901,13 @@ function conversation(App $a, $items, $mode, $update, $preview = false, $order =
  * @return array items with parents and comments
  */
 function conversation_add_children($parents, $block_authors, $order) {
-	$max_comments = Config::get("system", "max_comments", 100);
+	$max_comments = Config::get('system', 'max_comments', 100);
+
+	if ($max_comments > 0) {
+		$limit = ' LIMIT '.intval($max_comments + 1);
```

Comment:
Why + 1?

Reply:
This was part of the original coding. I guess the thought behind this is "X comments plus the original post". Yes, the original coding was from me as well, but several years ago.

Reply:
Sometimes documentation in the form of comments would make sense, especially at places like this one ;-)